lib/src/sv_common.c

```diff
@@ -109,6 +109,7 @@ bu_type_to_char(const bu_info_t *bu)
 
   switch (bu->bu_type) {
     case BU_TYPE_SEI:
+      if (bu->uuid_type == UUID_TYPE_ONVIF_MEDIA_SIGNING) return 'O';
```

Comment:
I don't think we should display to the user if it is a special ONVIF SEI. When validating with ONVIF Media Signing these characters will not be used anyhow.